game/core/src/me/lihq/game/people/NPC.java

```diff
@@ -192,6 +192,16 @@ public NPC setMotive(String motive)
     }
 
     /**
+     * This method sets the NPCs moveability to the value of the parameter
+     *
+     * @param canMove - whether or not to set allow the NPC to move
+     */
+    public void setCanMove(boolean canMove)
```

Comment:
Yeah in that case it makes sense, but when its just setting the value equal to what ever you put it, thats unnecessary